src/plugin/retail.js

```diff
@@ -34,9 +34,11 @@ Retail.prototype.setUpOnce = function(slave) {
 };
 
 Retail.prototype.onLoad = function(slave, loadData) {
-    loadData.data.settings.preventKeys.forEach(function(key) {
-        preventKeysList.push(key);
-    });
+    if (loadData.data.settings.hasOwnProperty('preventKeys')) {
+        loadData.data.settings.preventKeys.forEach(function(key) {
+            preventKeysList.push(key);
+        });
+    }
```

Comment:
This got me thinking: is there a reason why don't we just do here something like:

```javascript
preventKeys = loadData.data.settings.preventKeys || [];
```

instead of looping?